RecoTracker/LSTCore/standalone/code/core/write_lst_ntuple.cc

```diff
@@ -579,6 +621,146 @@ void setPixelTripletOutputBranches(lst::Event<Acc3D>* event) {
   ana.tx->setBranch<std::vector<int>>("pT3_isDuplicate", pT3_isDuplicate);
 }
 
+//________________________________________________________________________________________________________________________________
+void fillT5DNNBranches(lst::Event<Acc3D>* event, unsigned int T3) {
+  // Get relevant information
+  lst::Hits const* Hits = event->getHits()->data();
+  lst::Modules const* Modules = event->getModules()->data();
+
+  // Hits
+  std::vector<unsigned int> hits = getHitsFromT3(event, T3);
```

Comment:
try to come up with more creative names rather than `Hits` and `hits`; perhaps `hits` for the first and `hitIdx` for the latter